src/components/ChatLog.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  const chatComponents = props.entries.map((entry, index) => {
+    return (
+      <div key={index} className="chat-log">
+        <ChatEntry
+          id={entry.id}
+          sender={entry.sender}
+          body={entry.body}
+          timeStamp={entry.timeStamp}
+          liked={entry.liked}
+          colorLocal={props.colorLocal}
+          colorRemote={props.colorRemote}
+          onUpdate={props.onUpdateHeart}
+        />
+      </div>
+    );
+  });
+  return <section>{chatComponents};</section>;
```

Comment:
Style: Having the wrapper element of this be a `ul` or `ol` and the inner elements be `li`s would be more accessible.